exercises/practice/proverb/create_test_table.sql

```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS tests;
+CREATE TABLE IF NOT EXISTS tests (
+    -- uuid and description are taken from the test.toml file
+    uuid TEXT PRIMARY KEY,
+    description TEXT NOT NULL,
+    -- The following section is needed by the online test-runner
+    status TEXT DEFAULT 'fail',
+    message TEXT,
+    output TEXT,
+    test_code TEXT,
+    task_id INTEGER DEFAULT NULL,
+    -- Here are columns for the actual tests
+    strings TEXT NOT NULL,    -- json array
+    expected TEXT NOT NULL
+);
+
+INSERT INTO tests (uuid, description, strings, expected)
+    VALUES
```

Comment:
This is where the INSERT indentation comes from ;) The VALUES on its own line makes this consistent.